app/assets/stylesheets/admin/attachment.scss

```diff
@@ -61,46 +65,46 @@ $background-color: #002855;
 
 .submit-button {
   font-size: 20px;
-  width: 200px;
+  width: 30%;
   height: 40px;
   float: right;
-  background-color: $background-color;
-  border: 1px solid black;
+  background-color: $background-color-title;
+  border: 1px solid $black-color;
   border-radius: 20px;
-  color: #fff;
+  color: $white-color;
   cursor: pointer;
-  margin-right: 10px;
-  margin-bottom: 10px;
+  min-width: 80px;
 }
 
 .path-new-attachment {
   text-decoration: none;
-  color: grey;
-  background-color: $background-color;
+  color: $scrool-gray-color;
+  background-color: $background-color-title;
   border-radius: 5px;
   padding: 10px;
-  margin: 10px;
+  margin-left: 20%;
 }
 
 .path-attachment {
   margin: 20px;
-  margin-top: 20px;
 }
 
 .attachment-link {
   text-decoration: none;
-  color: black;
+  color: $black-color;
```

Comment:
I don't know why are you doing CSS variables? Do you have already skills how to use them?
I think yes, so start writing good CSS and good ruby code